components/form/FormInput.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from 'clsx'
+import { TextInput, View } from 'react-native'
+
+interface Props {
+  valueInput: string
```

Comment:
eliminar uso del nombre del componente en las props, se entiende que las props solamente afectan al componente FormInput, en su lugar : 
```suggestion
  value: string
  placeholder: string
```